shoop/admin/templates/shoop/admin/services/edit.jinja

```diff
@@ -43,17 +44,17 @@
 {{ super() }}
 <script>
     $(function () {
-        $("#component-button").on("click", function (e) {
-            e.preventDefault();
+        $("#component-button .btn").on("click", function (event) {
```

Comment:
Why this change? I think it's better to use the id straight at the clickable link element like it was...

Also using a class called `.btn` is not very good considering the previous version had only id selector which is unique and if someone decides to change the `<a>` classname to something different than btn, this stops working...